### PR TITLE
Add synphot to conda-forge

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwcraig** [comment]
@pllim  -- could you please confirm that you are willing to be a maintainer on this package?

**Comment by pllim** [comment]
@mwcraig , I hereby provide my consent being listed as a maintainer for this `synphot` recipe on `conda-forge`.

p.s. What do I have to do? It picks up PyPI release automatically, right?

File: recipes/synphot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c512ecd3dd2fb46830c8e99036f14e5bf03e8b624bd3c931e46676ff0a78b304
+  patches:
+    - setup.cfg.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```
Comment (xhochy) [inline recipes/synphot/meta.yaml]: The command can be simplified as the options are now part of the default.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Comment by mwcraig** [comment]
> p.s. What do I have to do? It picks up PyPI release automatically, right?

Ideally you won't need to do much 😀 . The bot opens a PR to update the package when a new version is pushed to pypi, so maybe merger those if I don't get to them promptly.

**Comment by pllim** [comment]
Will do. Thanks!

**Comment by pllim** [comment]
Thank you very much, @mwcraig ! As a follow-up, I am adding `stsynphot` in #8498 . Since that one is not Astropy-affiliated, no action is required on your part. This is just a FYI.

**Pull request merged**